src/main/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -38,6 +38,7 @@ public data class YamlConfiguration constructor(
     internal val strictMode: Boolean = true,
     internal val extensionDefinitionPrefix: String? = null,
     internal val polymorphismStyle: PolymorphismStyle = PolymorphismStyle.Tag,
+    internal val polymorphismPropertyName: String = "type",
```

Comment:
Could you please add a short explanation in the documentation above? Something like:

```
* [polymorphismPropertyName]: property name to use when [polymorphismStyle] is [PolymorphismStyle.Property]
```

Reply:
Thank you for accepting PR.

I added documentation comment.

I am happy to contribute to kaml project!